games-arcade/vvvvvv/vvvvvv-2.0~git.recipe

```diff
@@ -24,14 +24,14 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	libSDL2_2.0$secondaryArchSuffix
-	libSDL2_mixer_2.0$secondaryArchSuffix
+	libsdl2$secondaryArchSuffix
+	sdl2_mixer$secondaryArchSuffix
```

Comment:
Still don't provide the libs with "lib:"
```diff
-	libsdl2$secondaryArchSuffix
-	sdl2_mixer$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libSDL2_mixer_2.0$secondaryArchSuffix

Reply:
Tries these changes locally, but `haikuporter` fails to resolve the deps.